### PR TITLE
chore: move system extensions' udev rules

### PR DESCRIPTION
**Pull request by dsseng**
chore: move system extensions' udev rules

Ref: siderolabs/pkgs#1053

### CODE REVIEW

**Review by frezbo — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ TOOLS ?= ghcr.io/siderolabs/tools:v1.9.0-alpha.0-4-g2058296
 DEBUG_TOOLS_SOURCE := scratch
 
 PKGS_PREFIX ?= ghcr.io/siderolabs
-PKGS ?= v1.9.0-alpha.0-26-gc9282c8
+PKGS ?= v1.9.0-alpha.0-28-g38304a6
```
[the remark this answers is not in the document]
Comment (smira) [reply Makefile:25]: (or just remove this change)

**Review by smira — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -794,6 +794,7 @@ RUN <<END
 END
 
 FROM rootfs-base-${TARGETARCH} AS rootfs-base
+RUN mkdir -p /rootfs/usr/etc
```
Comment (frezbo) [inline Dockerfile:797]: https://github.com/dsseng/talos/blob/8680351c131d29a76682569742dbd44c8ffe47d3/Dockerfile#L701 this should have been here, can fixup later

Comment (dsseng) [reply Dockerfile:797]: Oops, will do. Thanks!

**Pull request merged**